main.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'app'
+
+class Main
+  def initialize
+    @app = App.new(self)
+  end
+
+  def display_menu
+    puts '---------------------------'
+    puts 'Welcome to the Library App!'
+    puts '---------------------------'
+    puts 'Please select an option:'
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
+    puts '4. Create a book'
+    puts '5. Create a rental'
+    puts '6. List rentals for a person'
+    puts '7. Quit'
+    puts '---------------------------'
+
+    choice = gets.chomp.to_i
+
+    case choice
+    when 1
+      list_all_books
+    when 2
+      list_all_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      list_rentals_for_person
+    when 7
+      quit_program
+    else
+      puts 'Invalid choice. Please try again.'
+      display_menu
+    end
+  end
```

Comment:
You did a good job working on this project! 💯 However, there are still some issues that need to be solved in order to continue to the next milestone! 

- [ ] Currently, when I try to run any option I'm getting this error:

![image](https://github.com/federicaulzurrun/SchoolLibrary/assets/77069011/9d8fff79-6a70-41bc-b226-8e1fd1a2d35a)

This is happening because these methods don't exist inside this class, they are instance methods of your `App` one. So please consider using `@app.method_name` instead! 🤓 🔧 

- [ ] Also, you have a typo in your `list_all_people` method. The actual name in your `app.rb` is `list_people`. Please consider fixing this! 🤓 🔧 

- [ ] The `quit_program` method doesn't exist. Please make sure to implement it! Currently, I'm getting this error:

![image](https://github.com/federicaulzurrun/SchoolLibrary/assets/77069011/52c47bfd-d035-4a8f-afbf-697666980462)

Reply:
@anagudelogu ohh I'm sorry i never checked again after fixing the linters problems